expansion/src/main/java/no/unit/nva/expansion/ExpansionConfig.java

```diff
@@ -8,6 +8,7 @@
 
     public static final Environment ENVIRONMENT = new Environment();
     public static final ObjectMapper objectMapper = JsonUtils.dtoObjectMapper;
+    public static final String ID_NAMESPACE = ENVIRONMENT.readEnv("ID_NAMESPACE");
```

Comment:
ID_NAMESPACE is used all over the place, so it is better to be in the common config class